sgv2-restapi/pom.xml

```diff
@@ -262,6 +262,46 @@
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
       </plugin>
+      <!-- Plugin for building Docker images -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>3.1.4</version>
+        <configuration>
+          <from>
+            <image>adoptopenjdk:8</image>
+          </from>
+          <to>
+            <image>docker.io/stargateio/stargate-restapi:${project.version}</image>
+          </to>
+          <!-- See: https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#container-object-->
+          <container>
+            <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
+            <filesModificationTime>${maven.build.timestamp}</filesModificationTime>
+            <ports>
+              <port>8082</port>
+            </ports>
+            <environment>
+              <STARGATE_GRPC_HOST>localhost</STARGATE_GRPC_HOST>
```

Comment:
I assume these are default values that may be overridden? (which makes sense)

Reply:
Yes exactly. I can put documentation into the README.